cypress/integration/publication_submission.spec.js

```diff
@@ -0,0 +1,38 @@
+describe('Publication: Description', () => {
+  beforeEach(() => {
+    cy.visit('/');
+    cy.server();
+  });
+
+  it('The user should be able to add and remove projects', () => {
+    cy.mocklogin();
+
+    const title = 'Sample title';
+    const description = 'Sample description.';
+
+    const isbn = '978-3-16-148410-0';
```

Comment:
https://www.mohrsiebeck.com/en/book/gesammelte-werke-in-deutscher-sprache-9783161484100?no_cache=1
beste boka

Reply:
ok. tar klagingen på alvor. fixer den....

Reply:
haha, var egentlig ikke klaging ble bare nysgjerrig på hva det var :p